src/pages/game/game.page.jsx

```diff
@@ -147,6 +206,32 @@ const GamePage = () => {
           </ion-fab-list>
         </ion-fab>
       </div>
+
+      {isChatDisplay && (
+        <ChatMessages
+          id={popOverId}
+          isOpen={isChatDisplay}
+          triggerId="chat"
+          handleNewChatMessageClick={handleNewChatMessageClick}
+          chatMessagesList={chatMessagesList}
+          open={open}
+          anchorEl={anchorEl}
+          onClose={handleClose}
+          chatBoxRef={chatBoxRef}
+        />
+      )}
+      <div className="floating-button">
+        <ion-fab horizontal="end" vertical="bottom" slot="fixed">
+          <ion-fab-button id="chat">
+            <ion-icon
+              name="chatbubbles-outline"
+              // onClick={(e) => setIsChatDisplay(!isChatDisplay)}
+              onClick={(e) => handleClick(e)}
```

Comment:
remove the commented onClick. Also move the current onClick to the ion-fab-button instead of the ion-icon. Currently, only if we press on the icon inside the fab button, the chat box opens, and not the whole button itself, which is a bad user experience.

Reply:
Oh ok, I got it. Changed.